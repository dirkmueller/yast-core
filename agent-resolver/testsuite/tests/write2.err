[YCP] tests/write2.ycp:3 .search true
[YCP] tests/write2.ycp:4 .nameservers true
[YCP] tests/write2.ycp:5 . true
