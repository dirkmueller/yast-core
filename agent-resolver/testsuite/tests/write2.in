domain suse.cz
nameserver 10.10.0.1
nameserver 10.10.128.1
search suse.de
