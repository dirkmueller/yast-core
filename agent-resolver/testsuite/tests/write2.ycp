{
    // y2warning(".domain %1", SCR::Write(.domain, nil));
    y2warning(".search %1", SCR::Write(.search, []));
    y2warning(".nameservers %1", SCR::Write(.nameserver, ["10.10.0.2", "10.10.128.2"]));
    y2warning(". %1", SCR::Write(., nil));
}
