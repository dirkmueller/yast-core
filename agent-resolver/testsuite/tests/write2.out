(nil)
domain suse.cz
nameserver 10.10.0.2
nameserver 10.10.128.2
